examples/candid_encoding/test/tests.ts

```diff
@@ -318,11 +219,13 @@ export function get_tests(
                     );
 
                 return {
-                    Ok:
-                        candid_encoded.Ok === true &&
-                        candid_decoded_result.includes('record') &&
-                        candid_decoded_result.includes('John') &&
-                        candid_decoded_result.includes('Doe')
+                    Ok: {
+                        passes:
```

Comment:
Rename to success

Reply:
done I did succeeds instead because a boolean value should be named as a true false question and not a verb

Reply:
well I guess in that case isSuccessful would be more correct

Reply:
Although looking at it closer its not always a boolean, its a generic T that just happens to be a boolean in every case

Reply:
So we get rid of the generic and rename it to isSuccessful